encoding/text/marshal.go

```diff
@@ -58,16 +58,22 @@ func (enc *Encoder) UseRegistry(reg *schemas.Registry) {
 	enc.nodes.UseRegistry(reg)
 }
 
+// SetIndent sets number of spaces to indent each level with.
+// A value of 0 disables indentation.
+func (enc *Encoder) SetIndent(indent int) {
```

Comment:
Rather than setting an integer number of spaces, this could use a string, which would allow indents with tabs, too.

Reply:
Done.